Automattic/woocommerce-payments
# Update/settings page ui

Related to #5372
Fix #5161

#### Changes proposed in this Pull Request
This PR updates the express checkouts ui so it follows the new designs from figma (C1r0Vzxh1mhbDJJnbjHp5q-fi-1437%3A71323)

#### Testing instructions

- Go to wp-admin > Payments > Settings >  Express Checkout and make sure the ui looks the same as the design in the figma file
- Change the setting and make sure they reflect in the product, cart and checkout pages. Also test the blocks pages.
-------------------


**Post merge**
